rust-runtime/aws-smithy-runtime-api/src/client/interceptors/error.rs

```diff
@@ -44,6 +44,7 @@ pub struct InterceptorError {
 
 impl InterceptorError {
     interceptor_error_fn!(read_before_execution => ReadBeforeExecution (with source));
+    interceptor_error_fn!(read_after_configuration => ReadAfterConfiguration (with source));
```

Comment:
did you still mean to add this error? I could be searching badly but I don't see it used

Reply:
Nope. I meant to remove this. Good catch!